benchmark/ubench.c

```diff
@@ -171,9 +155,17 @@ UBENCH_EX(simple, os_memory_provider) {
 
     umf_result_t umf_result;
     umf_memory_provider_handle_t os_memory_provider = NULL;
-    umf_result = umfMemoryProviderCreate(umfOsMemoryProviderOps(),
-                                         &UMF_OS_MEMORY_PROVIDER_PARAMS,
+    umf_os_memory_provider_params_handle_t os_params = NULL;
+
+    umf_result = umfOsMemoryProviderParamsCreate(&os_params);
+    if (umf_result != UMF_RESULT_SUCCESS) {
+        fprintf(stderr, "error: umfOsMemoryProviderParamsCreate() failed\n");
+        exit(-1);
+    }
+
+    umf_result = umfMemoryProviderCreate(umfOsMemoryProviderOps(), os_params,
                                          &os_memory_provider);
+    umfOsMemoryProviderParamsDestroy(os_params);
```

Comment:
we could also check return value of this func

Reply:
yeah, but what we should do with that? It does not impact the benchamrk. As you can see below we also do not check the return of the `umfMemoryProviderDestroy`